org/epifany/pkmnbreedprbltycalc/PkmnCmd.java

```diff
@@ -1,8 +1,10 @@
-package pokemoniv_4;
-
 /**
  * Copyright 2016, Stephen Gung, All rights reserved
- * 
+ */
+
+package org.epifany.pkmnbreedprbltycalc;
+
+/**
  * @author Stephen Gung
  */
 public class PkmnCmd{
```

Comment:
nit: space after class name and should use full words `PkmnCmd{` to `PokemonCommand {`

Reply:
That applies to all classes